addons/sourcemod/scripting/surftimer/commands.sp

```diff
@@ -359,6 +363,17 @@ public Action Command_ToggleHints(int client, int args)
 	return Plugin_Handled;
 }
 
+public Action Command_PRinfo_help(int client, int args){
+
+	if(IsValidClient(client)){
+		PrintToConsole(client, "%T", "PRinfo_help", client);
+		CPrintToChat(client, "%T", "PRinfo_help_chat", client, g_szChatPrefix);
```

Comment:
I believe that what @Bara meant was that this does not need the 2nd `client` variable as in the translation you are only passing the `g_szChatPrefix` at the start of the message and nothing else.

Also is there a difference between `%T` and `%t` ? Just looking that all the other translation files have the `%t` format might be worth keeping to that unless something is different when using capital letter for it.

Reply:
%t uses the server language (mostly english) and %T uses the client language which would be more useful (my opinion). That's why you need after the Phrase Name the client index. So the second client is required using %T.

More informations are here about this: https://wiki.alliedmods.net/Translations_(SourceMod_Scripting)#Usage_in_a_Plugin